dev/NavigationView/NavigationView.idl

```diff
@@ -159,6 +159,7 @@ unsealed runtimeclass NavigationView : Windows.UI.Xaml.Controls.ContentControl
     [MUX_DEFAULT_VALUE("1008.0")]
     [MUX_PROPERTY_VALIDATION_CALLBACK("CoerceToGreaterThanZero")]
     Double ExpandedModeThresholdWidth { get; set; };
+    Windows.Foundation.Collections.IVector<Object> FooterMenuItems{ get; };
```

Comment:
No exposed setter?

Reply:
Why it should? This is collection.
Menu items has same declaration. 
`Windows.Foundation.Collections.IVector<Object> MenuItems { get; };`